app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogFragment.kt

```diff
@@ -129,6 +129,9 @@ class AppRatingDialogFragment : DialogFragment() {
         if (!TextUtils.isEmpty(description)) {
             dialogView.setDescriptionText(description!!)
         }
+        if (!TextUtils.isEmpty(defaultComment)) {
```

Comment:
You could rely on Kotlin only here i.e.
```
!TextUtils.isEmpty(defaultComment)
```
->
```
!defaultComment.isNullOrEmpty()
```

Reply:
thanks for advice :) corrected